static/src/javascripts/projects/common/modules/experiments/tests/ias-ad-targeting.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+export const iasAdTargeting: ABTest = {
+    id: 'IasAdTargeting',
+    start: '2017-10-31',
+    expiry: '2017-10-04',
```

Comment:
Is it me, or does this test end before it begins? 😄

Reply:
I was just checking the AB build tests, you know, for science and rigour...

Reply:
A very admirable endeavour! 😆